src/data/wallets/wallet-data.ts

```diff
@@ -44,7 +44,8 @@ export const walletsData: WalletData[] = [
     last_updated: "2022-06-22",
     name: "Keystone",
     image: KeystoneImage,
-    brand_color: "#ffffff",
+    color: "bg-[#FFFFFF]",
+    brand_color: "from-[#ffffff]",
```

Comment:
Not blocking: ideally, these files should know nothing about the implementation details to keep them decoupled. I'd have left the raw colors and applied them using `style={{ background-color: ${...} }}`

Reply:
Okay, so @pettinarip while working within the confines of Tailwinds, you're proposing that in this case where we would need to string interpolate to just apply it using `style`...

Just want to note, that while `bg-[#FFFFFF]` is equivilent to `background-color: #FFFFFF`:

```css
.bg-\[\#FFFFFF\] {
  --tw-bg-opacity: 1;
  background-color: rgb(255 255 255 / var(--tw-bg-opacity)) /* #ffffff */;
}
```

...worth noting that `from-[#ffffff]` is actually altering a CSS variable `--tw-gradient-from`:

```css
.from-\[\#ffffff\] {
  --tw-gradient-from: #ffffff var(--tw-gradient-from-position);
  --tw-gradient-to: rgb(0 0 0 / 0) var(--tw-gradient-to-position);
  --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to);
}
```

When I adjust this inside the `style` attribute I get a type error if I don't add `as React.CSSProperties`:

```
You can't pass property "--tw-gradient-from" to type Properties<string | number, string & {}>.
```

```tsx
  style={{
    "--tw-gradient-from": "#000000",
  } as React.CSSProperties}
```

Reply:
Similar, don't see as a blocker but I do tend to agree that the property name should indicate what it is... so if we're calling it `color`, we should pass a color, not a tw class... alternatively we could rename the property to `twBgClass` for example to more accurately explain what is to be expected. Would patch separately to avoid adding blockers.

Reply:
Ah I didn't see that `.from-[...]` case. Yeah, I would expect to do something like you shared. The type error I think is fine since it is not a known css prop name, but curious, did that work?

Again, minor thing, we can skip this. I like the renaming idea as well but not a strong stance from my side.

Reply:
Hm.. actually, no, not as we'd like it. So, we can declare that variable, but without having a TW class in there to tee-up the other parts of that class, it doesn't work:

```css
.from-\[\#ffffff\] {
  --tw-gradient-from: #ffffff var(--tw-gradient-from-position);
  --tw-gradient-to: rgb(0 0 0 / 0) var(--tw-gradient-to-position);
  --tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to);
}
```

`--tw-gradient-from` and `--tw-gradient-to` are instantiated with the `.from-[<color>]` class, but more importantly they do nothing without the `--tw-gradient-stops: var(--tw-gradient-from), var(--tw-gradient-to);` also found in that TW class.

A workaround is to declare `from-transparent` and then _also_ apply the `style={{"--tw-gradient-from": "#000000"} as React.CSSProperties}` override... this will set up the TW gradient CSS variables, and then we overwrite the `from` token

It's hacky imo... I'm personally leaning towards just updating the variable _names_ to say `textClass` and `bgClass` instead of `color` and `brand_color`.

Reply:
Also just noting for learning purposes that the `to-[]` or `bg-gradient-to-[]` class does _not_ tee up `--tw-gradient-stops`; only `from-[]`

Reply:
Will update the variable name here in data :)